dll/win32/msv1_0/msv1_0.c

```diff
@@ -1478,14 +1485,17 @@ LsaApLogonUserEx2(IN PLSA_CLIENT_REQUEST ClientRequest,
             }
 
             /* Check if the account expired */
-            AccountExpires.LowPart = UserInfo->All.AccountExpires.LowPart;
-            AccountExpires.HighPart = UserInfo->All.AccountExpires.HighPart;
-            if (LogonTime.QuadPart >= AccountExpires.QuadPart)
+            if (!(UserInfo->All.UserAccountControl & USER_DONT_EXPIRE_PASSWORD))
```

Comment:
This seems like it belongs in a separate commit?

Reply:
Reverted. Will be a new PR/commit.